dev_docs/tutorials/building_a_kibana_distributable.mdx

```diff
@@ -0,0 +1,135 @@
+---
+id: kibDevTutorialBuildingDistributable
+slug: /kibana-dev-docs/tutorial/building-distributable
+title: Building a Kibana distributable
+summary: Learn how to build a Kibana distributable
+date: 2021-05-10
+tags: ['kibana', 'onboarding', 'dev', 'tutorials', 'build', 'distributable']
+---
+
+On the course of that tutorial it will be possible to learn more about how we can create a Kibana distributable 
+as well as handle its different main configurations.
+
+At any given point, you can get CLI help running the following command:
+
+```bash
+yarn build --help
+```
+
+## Prerequisites
+
+For the basic steps of the build we only require you to use one of the following operating systems:
+
+- Linux
+- macOS
+
+However if as part of the build you also want to generate Linux installation packages (deb, rpm) or Docker images there 
+are other dependencies to have in mind as those installation packages and docker images generation are run using fpm, dpkg, rpm, and Docker.
+
+### For Linux Installation Packages Build
+
+Please make sure you have the following dependencies installed:
+
+#### Install FPM dependencies
+
+**On OSX/macOS:**
+
+```bash
+brew install gnu-tar
```

Comment:
Do we need the rpm and dpkg packages here?

Reply:
I've followed what is described in the official documentation for fpm at https://fpm.readthedocs.io/en/latest/installing.html

Reply:
I don't have a macOS machine to test - but if this works LGTM

Reply:
@jbudz I reconfirmed this and we need `rpm` to be installed along brew. It only worked before because I already had rpm installed. With that change I think we are 100% good here.